### PR TITLE
Fix test coverage for resource event

### PR DESCRIPTION
**Pull request by leplatrem**
The whole initialization code and resource event subscriber was not tested at all.

This fixes it :)

@Natim r?

### CODE REVIEW

**Comment by Natim** [comment]
r+

File: kinto_signer/__init__.py [hunk 1]
```diff
@@ -1,2 +1,73 @@
-from kinto_signer.updater import LocalUpdater  # noqa
-from kinto_signer.hook import includeme  # noqa
+import functools
```
Comment (almet) [inline kinto_signer/__init__.py]: I put this in the `hook` module on purpose, to do the imports in `__init__.py`. What's the value of moving this in the same module?

File: kinto_signer/tests/test_plugin_setup.py [hunk 2]
```diff
@@ -13,7 +21,7 @@ def test_capability_is_exposed(self):
         capabilities = resp.json['capabilities']
         self.assertIn('signer', capabilities)
         expected = {
-            "description": "Provide signing capabilities to the server.",
+            "description": "Digital signatures for integrity and authenticity of records.",  # NOQA
```
Comment (almet) [inline kinto_signer/tests/test_plugin_setup.py]: of collections

**Comment by almet** [comment]
Thanks for adding these tests :)

File: kinto_signer/__init__.py (hunk 1 above)
Comment (leplatrem) [inline kinto_signer/__init__.py]: What do you mean by `to do the imports in __init__.py` ?

I wanted to keep it simple and stupid, less files, includeme implemented like in other plugins.

File: kinto_signer/tests/test_plugin_setup.py (hunk 2 above)
Comment (leplatrem) [inline kinto_signer/tests/test_plugin_setup.py]: of collections of records ?

File: kinto_signer/__init__.py (hunk 1 above)
Comment (almet) [inline kinto_signer/__init__.py]: I mean: keep the `__init__` file as clean as possible, and only do the imports in this file.

As I won't maintain this project, I wont object any further. Just wanted to state why I did it this way.

File: kinto_signer/tests/test_plugin_setup.py (hunk 2 above)
Comment (almet) [inline kinto_signer/tests/test_plugin_setup.py]: Is there collections of something else than records?

Comment (leplatrem) [inline kinto_signer/tests/test_plugin_setup.py]: But in the end, the integrity and authenticity is about records ?

Comment (almet) [inline kinto_signer/tests/test_plugin_setup.py]: Let's stop playing on words, it doesn't add much value :) Either records or collection are okay fro mmy perspective, sorry for the noise.

File: kinto_signer/__init__.py (hunk 1 above)
Comment (leplatrem) [inline kinto_signer/__init__.py]: Keeping the `__init__` as empty as possible is not a feature IMO :)

Especially when it comes to see this kind of things in configuration files:

capture d ecran de 2016-04-25 11-53-37

**Pull request merged**